packages/shared/src/api/dto/UpdateTaskCommand.dto.ts

```diff
@@ -0,0 +1,9 @@
+export interface UpdateTaskCommandDTO {
```

Comment:
rename to "UpdateTaskCommand" and move to: packages/shared/src/api/command

Reply:
Done